### PR TITLE
remove unused module Limit.hs

### PR DESCRIPTION
**Pull request by DeepakKapiswe**
addresses #24

### CODE REVIEW

**Comment by ruhatch** [comment]
I think this is used in the `cardano-crypto-wrapper` package...could you confirm that please

**Comment by DeepakKapiswe** [comment]
yes, Limit is used at several places in cardano-ledger.

**Comment by DeepakKapiswe** [comment]
Cardano-ledger uses Limit Type at several places so we can't remove it, although it is not used in Cardano-binary.

**Comment by erikd** [comment]
> yes, Limit is used at several places in cardano-ledger.

Its seems the only place `Limits` or any of the associated stuff is used in in the `Cardano.Crypto.Limits` modules and that in turn is only used in tests. Nowhere else in this new code base is `Cardano.Binary.Limits` or `Cardano.Crypto.Limits` used.

**Comment by ruhatch** [comment]
Aha, so we should move it to `Test.Cardano.Binary.Limits`?

**Comment by DeepakKapiswe** [comment]
Limit type is used only at `Test.Cardano.Crypto.Limits` so I think we can drop the Limit.hs module here.

File: binary/test/Test/Cardano/Binary/Helpers.hs [hunk 1]
```diff
@@ -179,72 +168,6 @@ extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
     (encoded :: a) = unsafeDeserialize (serialize u)   -- Step 3
   in encoded === input
 
---------------------------------------------------------------------------------
--- Message length
---------------------------------------------------------------------------------
-
-msgLenLimitedCheck :: ToCBOR a => Limit a -> a -> Property
```
Comment (ruhatch) [inline binary/test/Test/Cardano/Binary/Helpers.hs]: Can you check whether this stuff has been translated into `hedgehog` tests in `cardano-crypto-tests`? We should have these tests, even if they're currently unused.

Comment (DeepakKapiswe) [reply binary/test/Test/Cardano/Binary/Helpers.hs]: https://github.com/input-output-hk/cardano-ledger/blob/e2d4663f2251ef76bae94e48fef910b8f5f867bd/crypto/test/Test/Cardano/Crypto/Limits.hs#L59

**Pull request merged**